packages/arb-token-bridge-ui/src/hooks/useTransactions.ts

```diff
@@ -64,11 +64,20 @@ export const txnTypeToLayer = (txnType: TxnType): 1 | 2 => {
 
 export interface L1ToL2MessageData {
   status: L1ToL2MessageStatus
-  retryableCreationTxID: string
+  retryableCreationTxID?: string
   l2TxID?: string
   fetchingUpdate: boolean
 }
 
+export interface L2ToL3MessageData {
+  status: L1ToL2MessageStatus
+  retryableCreationTxID?: string
+  l3TxID?: string
+  fetchingUpdate: boolean
+  l2ChainId: number
+  completed: boolean
+}
```

Comment:
To simplify things, I have removed `fetchingUpdate` and `completed` from `L2ToL3MessageData` since we don't use them anywhere, but we will definitely need `L2ToL3MessageData (or something similar)` in conjunction with the existing `L1ToL2MessageData` to hold information about the second retryable in the tx.